lib/models/storage/member_service/property_storage_service.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:hatspace/data/data.dart';
+
+class PropertyService {
+  final FirebaseFirestore _firestore;
+  final String propertyCollection = 'properties';
+
+  PropertyService(FirebaseFirestore firestore) : _firestore = firestore;
+
+  Future<Property?> getProperty(String id) async {
+    DocumentSnapshot<Map<String, dynamic>> proppertiesRef = await _firestore
+        .collection(propertyCollection)
+        .doc(id)
+        .get(const GetOptions(source: Source.server));
```

Comment:
Using Source.server will not work when device does not have internet connection. We'll need to handle how app behaves when user does not have internet connection. Let's talk to PO about this.